set.mm

```diff
@@ -99022,59 +99022,19 @@ our Axiom of Choice (in the form of ~ ac2 ).  The proof does not depend
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
   Cardinal number arithmetic
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-$)
-
-  $c +c $. $( Cardinal number addition $)
-
-  $( Extend class definition to include cardinal number addition. $)
-  ccda $a class +c $.
 
-  ${
-    $d x y $.
-    $( Define cardinal number addition.  Definition of cardinal sum in
-       [Mendelson] p. 258.  See ~ cdaval for its value and a description.
-       (Contributed by NM, 24-Sep-2004.)  Use ~ df-dju instead.
-       (New usage is discouraged.) $)
-    df-cda $a |- +c = ( x e. _V , y e. _V |->
-                   ( ( x X. { (/) } ) u. ( y X. { 1o } ) ) ) $.
-  $}
+  For cardinal arithmetic, we follow [Mendelson] p. 258.  Rather than
+  defining operations restricted to cardinal numbers, we use Cartesian
+  product and set exponentiation for cardinal multiplication and
+  exponentiation.  We use the disjoint union ~ df-dju ( ` |_| ` )
+  for cardinal addition.  Equinumerosity and dominance serve the roles
```

Comment:
It's a bit strange to introduce multiplication, then exponentiation, then addition. It would be more natural to list them with addition first, and more uniformly, i.e. if we reference df-dju, we also reference the definitions for the other two.

Reply:
Yeah the wording was always a bit odd but I guess it gets even worse with this not being a comment on cdaval .

I've updated it as you suggest.